src/pylint_odoo/checkers/odoo_addons.py

```diff
@@ -170,6 +170,7 @@
         CHECK_DESCRIPTION,
     ),
     "E8130": ("Test folder imported in module %s", "test-folder-imported", CHECK_DESCRIPTION),
+    "E8135": ("Compute method calling 'write' method", "compute-write", CHECK_DESCRIPTION),
```

Comment:
@pedrobaeza @luisg123v 

Could you help me to define a better and more intuitive message and check-name here, please?

Reply:
I think this can serve:

`"Compute method calling 'write'. Use 'update' instead."`

Reply:
Good!

Thank you!

And name-of-check?

Reply:
`no-write-in-compute`?